dbus_service.py

```diff
@@ -291,8 +291,7 @@ def _get_status_url(self):
 
     def get_opendtu_base_url(self):
```

Comment:
What do you mean by "dtu element"? I removed the username and password here already. :-)

Reply:
I assume, he hadn't read your new changes before writing. So he is talking about what you did and everything is fine :-)